keps/sig-scheduling/3243-respect-pod-topology-spread-after-rolling-upgrades/README.md

```diff
@@ -640,7 +640,13 @@ feature gate after having objects written with the new field) are also critical.
 You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
-No, unit and integration tests will be added.
+Yes. there are unit and integration tests for feature enablement/disablement.
+- unit tests:
+  - `pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go`
+  - `pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go`
+- integration tests
+  -  `test/integration/scheduler/filters/filters_test.go`
+  -  `test/integration/scheduler/scoring/priorities_test.go`
```

Comment:
These are all good tests that are testing the feature, but they don't test "enablement/disablement".

I would like to see a test that is exercising the enablement/disablement the feature - see the comment above for an example of such test:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282

[basically a unit test that changes the value of the feature gate in the middle of the test]

Reply:
Updated